src/collective/ifttt/utils.py

```diff
@@ -84,7 +87,8 @@ def configure_rule(self, data):
         self.rule_id = storage.values()[-1].id
 
         # traverse to configuration page of content rule
-        rule_url = '/Plone/' + self.rule_id
+        portal = api.portal.get().__name__
```

Comment:
Interesting that you found `__name__` for `api.portal.get().__name__`. More usual one found in existing code is `api.portal.get().getId()`.